user_guide_src/source/helpers/html_helper.rst

```diff
@@ -253,52 +251,67 @@ The following functions are available:
 .. php:function:: audio($src[, $unsupportedMessage = ''[, $attributes = ''[, $tracks = [][, $indexPage = false]]]])
 
     :param  mixed   $src:                Either a source string or an array of sources. See :php:func:`source()` function
-    :param  string  $unsupportedMessage: The message to display if the media tag is not supported by the browser
-    :param  string  $attributes:
+    :param  string  $unsupportedMessage: The message to display if the audie element is not supported by the browser
+    :param  string  $attributes:         HTML attributes
     :param  array   $tracks:             Use the track function inside an array. See :php:func:`track()` function
-    :param  bool    $indexPage:
-    :returns:                            HTML-formatted audio element
+    :param  bool    $indexPage:          Should indexPage be added to the audio source path
+    :returns:                            An HTML audio element
     :rtype: string
 
-    Identical to :php:func:`video()`, only it produces the ``<audio>`` tag instead of ``<video>``.
+    Identical to :php:func:`video()`, only it produces ``<audio>`` element instead of ``<video>``.
 
-.. php:function:: source($src = ''[, $type = false[, $attributes = '']])
+.. php:function:: source($src[, $type = false[, $attributes = '']])
 
     :param  string  $src:        The path of the media resource
     :param  bool    $type:       The MIME-type of the resource with optional codecs parameters
-    :param  array   $attributes: HTML attributes
-    :returns:   HTML source tag
+    :param  string  $attributes: HTML attributes
+    :returns:                    An HTML source element
     :rtype: string
 
-    Lets you create HTML ``<source />`` tags. The first parameter contains the
-    source source. Example:
+    Lets you create HTML ``<source>`` elements. The first parameter contains the path of the resource.
+    Example:
 
     .. literalinclude:: html_helper/015.php
 
-.. php:function:: embed($src = ''[, $type = false[, $attributes = ''[, $indexPage = false]]])
+.. php:function:: track($src, $kind, $srcLanguage, $label)
+
+    :param  string $src:         The path of the media resource
+    :param  string $kind:        The kind of timed track
+    :param  string $srcLanguage: The language of the timed track
+    :param  string $label:       A user-readable title for the timed track
+    :returns:                    An HTML track element
+    :rtype: string
+
+    Generates a track element to specify timed tracks. The tracks are
+    formatted in WebVTT format. Example:
+
+    .. literalinclude:: html_helper/019.php
```

Comment:
These differences make it difficult to review.
I don't know why `track()` location was changed.
Please create one commit for changing the position.

Also, there are too many diffs in one commit. 
Many fixes are corrections of errors, and it is too large to be called typos.
Please create a commit with a commit log of what you changed for every single meaningful fix.